roles/validate-app/tasks/main.yaml

```diff
@@ -8,32 +8,29 @@
       command: "{{ openshift.common.client_binary }} new-project validate"
```

Comment:
I'd say to set validate to a variable to avoid rewritting it a few times (and change it if needed)